contracts/StorageManager.sol

```diff
@@ -192,16 +234,15 @@ contract StorageManager {
 
         // Allow to enforce payout funds and close of agreements that are already expired,
         // which should free needed capacity, if the capacity is becoming depleted.
-        if (agreementsReferencesToBePayedOut.length > 0) {
-            _payoutFunds(agreementsReferencesToBePayedOut, payable(provider));
+        if (dataReferencesOfAgreementToPayOut.length > 0) {
+            _payoutFunds(dataReferencesOfAgreementToPayOut, creatorsOfAgreementToPayOut, tokenOfAgreementsToPayOut, payable(provider));
```

Comment:
Move this one up into the function, before we are doing any accounting/check on our own agreement.

Reply:
Done